src/main/java/core/basesyntax/Colors.java

```diff
@@ -0,0 +1,9 @@
+package core.basesyntax;
+
+public enum Colors {
```

Comment:
```suggestion
public enum Color {
```
Its better to use singular form for enum naming